vmServer/send/proto/Request.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+message TaskRequest {
+  string codePath=1;    //  pantheon path where the input code is stored
```

Comment:
nit: spacing - have a white-space before and after the equals. (please do the same in the rest of the file as well)

Reply:
I think we should follow https://developers.google.com/protocol-buffers/docs/style#message_and_field_names for naming, specifically use "code_path" instead of "codePath"

Reply:
Space around operators added and changed the naming to follow snake_case style even in proto fields.